### PR TITLE
Disable logging when verbosity=0

### PR DESCRIPTION
**Pull request by abhi1693**
Depends on #414

### CODE REVIEW

File: daphne/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import sys
 
-__version__ = "3.0.2"
+__version__ = "3.0.4"
```
Comment (carltongibson) [inline daphne/__init__.py:3]: Please drop this.

File: daphne/cli.py [hunk 2]
```diff
@@ -90,6 +90,11 @@ def __init__(self):
             help="Where to write the access log (- for stdout, the default for verbosity=1)",
             default=None,
         )
+        self.parser.add_argument(
+            "--log-fmt",
+            help="Log format to use",
+            default="%(asctime)-15s %(levelname)-8s %(message)s"
+        )
```
Comment (carltongibson) [inline daphne/cli.py:97]: Drop this.

File: daphne/cli.py [hunk 3]
```diff
@@ -208,15 +213,16 @@ def run(self, args):
         # Decode args
         args = self.parser.parse_args(args)
         # Set up logging
-        logging.basicConfig(
-            level={
-                0: logging.WARN,
```
Comment (carltongibson) [inline daphne/cli.py:213]: This is a change in behaviour... `verbosity=0` leads to `logging.WARN` currently.

**Review by carltongibson — COMMENTED** [review]
I suspect we don't want this. Disabling logging entirely, rather than setting a high level, isn't the usual approach. If you wanted that you could use `disable_existing_loggers` in your [logging config](https://docs.python.org/3/library/logging.config.html#configuration-dictionary-schema) (for example)

I'm going to close this for now. Happy to discuss it some more though.

**Comment by abhi1693** [comment]
@carltongibson I already have `disable_existing_loggers=True` and yet `daphne` prints my JSON structured logs again to the console as JSON string along with the original log. With verbosity=0, with my changes, I was able to fix this to avoid invalid console prints.

Would it make sense to set `verbosity=-1` then?

Current behaviour

```
{"request_id": "10a9e8a85d4e7804f8971bdb82e69d2b", "user_id": null, "ip": "x.x.x.x", "event": "Testing loggers", "timestamp": "2022-04-11T03:37:53.900919Z", "logger": "oms.base", "level": "info"}
2022-04-11 03:37:53,900 INFO     {'request_id': '10a9e8a85d4e7804f8971bdb82e69d2b', 'user_id': None, 'ip': 'x.x.x.x', 'event': 'Testing loggers', 'timestamp': '2022-04-11T03:37:53.900919Z', 'logger': 'oms.base', 'level': 'info'}
```

Expected behaviour:

```
{"request_id": "10a9e8a85d4e7804f8971bdb82e69d2b", "user_id": null, "ip": "x.x.x.x", "event": "Testing loggers", "timestamp": "2022-04-11T03:37:53.900919Z", "logger": "oms.base", "level": "info"}
```

**Pull request closed without merging**